AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -168,6 +171,11 @@ - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionVi
   
   [self registerClass:[UICollectionViewCell class] forCellWithReuseIdentifier:@"_ASCollectionViewCell"];
   
+  if (ASSystemVersionLessThan8()) {
```

Comment:
Using the math term "less" is harder than the temporal terms "Before" / "After".  Maybe ASSystemVersionBefore8()

Perhaps even better, ASSystemVersionIsBefore(8.0) ?